apis/v1beta1/validation/gateway.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2021 The Kubernetes Authors.
+Copyright 2023 The Kubernetes Authors.
```

Comment:
Generally this date represents when the file was created, not when it was updated, I'd leave it unchanged.
```suggestion
Copyright 2021 The Kubernetes Authors.
```

Reply:
Fixed.